src/components/interaction-designer/block-editors/GenericContactPropertyEditor.vue

```diff
@@ -150,16 +120,16 @@ export class GenericContactPropertyEditor extends mixins(Lang) {
     this.shouldSetContactProperty = has(this.block.config, 'set_contact_property')
     this.propertyKey = this.firstContactPropertyKey
     this.propertyValue = this.firstContactPropertyValue
-    if (this.propertyValue === undefined) {
-      // default setting
-      if (this.isBlockInteractive(this.block)) {
-        // interactive blocks will have `Entry from this block` option by default
-        this.propertyValue = BLOCK_RESPONSE_EXPRESSION
-      } else {
-        // non interactive blocks will have `Expression` option by default
-        this.propertyValue = EMPTY_STRING_EXPRESSION
-      }
+
+    // default setting
+    if (this.isBlockInteractive(this.block)) {
+      // interactive blocks will have `Entry from this block` option by default
+      this.propertyValue = BLOCK_RESPONSE_EXPRESSION
+    } else {
+      // non interactive blocks will have `Expression` option by default
+      this.propertyValue = EMPTY_STRING_EXPRESSION
     }
+
     this.initPropertyValueAction()
   }
```

Comment:
Could you comment a bit on the changes in this file, please? 😉

Reply:
I mean, I'm afraid that removing stuff from the template will affect other blocks that use GenericContactPropertyEditor, like the Numeric block, won't it?
<img width="354" alt="Screen Shot 2022-09-02 at 12 55 27 PM" src="https://user-images.githubusercontent.com/18130716/188125163-d14cc130-bde6-46bc-83fd-e6880b8f94ba.png">